generator/svelte/src/lib/components/app/MasjidsList.svelte

```diff
@@ -36,8 +38,14 @@
 						</thead>
 						<tbody>
 							{#each entries(iqamas) as [iqama, { time, changed_on: changedOn }]}
-								<tr>
-									<td class="capitalize">{iqama}</td>
+								<tr class={tw(['!bg-primary/5 font-semibold', isNextIqama(masjids, name, iqama)])}>
```

Comment:
The alternating white/grey does not look good with the special background for the next prayer. Knowing that all masjids will have a next prayer (did you test if time is after Isha so next is fajr?), I recommend removing the grey so that all rows will be white (with line separator) leaving visual room for the next prayer to have a different background.

Reply:
Yes, we could remove the zebra table. It's all temporary though, we will start enhancing the UI soon by studying it in Figma. 

The next prayer is a linked list implementation so every prayer has a next one, I didn't test it well though.

Reply:
@hammady Screenshots updated

Reply:
@hammady Doesn't look too good I know. It's over simplified. but don't worry, we will go through this soon :D

Reply:
I like its simplicity! If we are to do another iteration in the frontend, it will be a mobile app :)